ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -97,10 +98,12 @@ public void setRespiratoryField(String value) {
     }
 
     public void clearPatientBloodPressure1() {
+    	waiter.until(ExpectedConditions.visibilityOfElementLocated(BLOOD_PRESSURE_FIELD_1));
         findElement(BLOOD_PRESSURE_FIELD_1).clear();
     }
   
     public void clearPatientBloodPressure2() {
+    	waiter.until(ExpectedConditions.visibilityOfElementLocated(BLOOD_PRESSURE_FIELD_2));
```

Comment:
```suggestion
        waiter.until(ExpectedConditions.visibilityOfElementLocated(BLOOD_PRESSURE_FIELD_2));
```

Reply:
Hey @ibacher  , Didnt understand this commit, it might not follow this standard